src/panels/lovelace/entity-rows/hui-media-player-entity-row.js

```diff
@@ -103,22 +103,25 @@ class HuiMediaPlayerEntityRow extends LocalizeMixin(PolymerElement) {
   _computeMediaTitle(stateObj) {
     if (!stateObj || this._isOff(stateObj.state)) return null;
 
+    let prefix =
```

Comment:
I dont think we should define `suffix` and `prefix` before the switch statement if they are just getting re-written anyway. 
```suggestion
    let prefix, suffix;
    switch (stateObj.attributes.media_content_type) {
        case "music":
            prefix = stateObj.attributes.media_artist;
            suffix = stateObj.attributes.media_title;
            break;
        case "tvshow":
            prefix = stateObj.attributes.media_series_title;
            suffix = stateObj.attributes.media_title;
            break;
        default:
            prefix = stateObj.attributes.media_title ||
                     stateObj.attributes.app_name ||
                     stateObj.state;
            suffix = "";
```

Reply:
Looks like the `one-var` rule is enabled in ESLint, splitting your `let` declaration into two lines. Apart from that change (and some prettier formatting) I've applied your suggestion.